progressmeter.c

```diff
@@ -160,11 +163,25 @@ refresh_progress_meter(void)
 	buf[0] = '\0';
 	file_len = win_size - 35;
 	if (file_len > 0) {
+#ifdef IRONCORE
+#define LOCK_PREFIX_EXTRA_CHARS 4	//  For the return character and the first 3 of 4 bytes of the lock icon,
+	   								//	none of which take any space on the screen.
+		if (iron_extension_offset(file) >= 0) {
```

Comment:
If the file name consists of nothing more than the `.iron` extension do you want to go down this path? I really have no idea what the right thing to do in that case would be, but it certainly seems degenerate.